plugins/database/src/db_plugin.cpp

```diff
@@ -14499,7 +14507,20 @@ irods::error db_mod_ticket_op(
                     log_sql::debug("chlModTicket SQL 11");
                 }
 
-                return execute_sql("insert into R_TICKET_ALLOWED_HOSTS (ticket_id, host) values (? , ?)");
+                irods::error ret = execute_sql("insert into R_TICKET_ALLOWED_HOSTS (ticket_id, host) values (? , ?)");
+                if (!ret.status()) {
```

Comment:
I believe we normally use `if (!ret.ok())`, why `.status()`?

_This applies to the other locations too._

Reply:
I had noticed in `irods_error.cpp:257` that there was a comment saying `irods::error::ok()` was deprecated in 4.0.3. If this is not the case I will change my code to use `irods::error::ok()` as I believe it is clearer.

Reply:
I was not aware of that. We still use it afaik.

Let's stick with `.ok()` for now.